src/eval.ml

```diff
@@ -108,9 +112,12 @@ and whnf_stk : term -> stack -> term * stack = fun t stk ->
   (* In head normal form. *)
   | (_         , _    ) -> st
 
-(** [eq_modulo a b] tests equality modulo rewriting between [a] and [b]. *)
-and eq_modulo : term -> term -> bool = fun a b ->
-  if !log_enabled then log_conv "[%a] == [%a]" pp a pp b;
+(** [eq_modulo ?ctx a b] tests equality modulo rewriting and modulo Δ
+    (unfolding of variables from the context) between [a] and [b] and with the
+    context [?ctx] if provided. *)
+and eq_modulo : ?ctx:ctxt -> term -> term -> bool =
```

Comment:
please replace `?ctx:ctxt` by `ctxt` (we can also give the empty context as argument if we don't care)